include/lug/System/Memory/Allocator/Chunk.inl

```diff
@@ -0,0 +1,38 @@
+template <size_t MaxSize, size_t MaxAlignment, size_t Offset>
+Chunk<MaxSize, MaxAlignment, Offset>::Chunk(lug::System::Memory::Area::IArea* area) : _area{area}, _currentPage{_area->requestNextPage()}, _firstPage{_currentPage} {}
+
+template <size_t MaxSize, size_t MaxAlignment, size_t Offset>
+void* Chunk<MaxSize, MaxAlignment, Offset>::allocate(size_t size, size_t alignment, size_t offset) {
+    LUG_ASSERT(offset == Offset, "Chunk allocator doesn't support multiple offset");
+    LUG_ASSERT(MaxSize >= size, "Size of the allocation is greater than the chunk max size");
+    LUG_ASSERT(MaxAlignment >= alignment, "Alignment of the allocation is greater than the chunk max alignment");
```

Comment:
chunk's max alignment

(maybe, I don't know, add the value of the alignment that was exceeded in the message)